docs/.vitepress/config.ts

```diff
@@ -0,0 +1,66 @@
+const utils = require('./utils.ts');
+
+utils.copyFile('README.md', 'index.md');
+utils.copyFile('CONTRIBUTING.md', 'CONTRIBUTING.md');
+
+utils.rewriteFile('../apidocs/README.md', /\(CONTRIBUTING.md\)/g, '(../CONTRIBUTING.md)');
```

Comment:
typedoc copies the `/README.md` to `docs/apidocs/README.md`.  And it contains the link to `CONTRIBUTING.md`. But there's no document at `docs/apidocs/CONTRIBUTING.md`. We need to fix the link.